packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -296,7 +297,8 @@ export const workflowsRouter = createProtectedRouter()
         if (
           newEventType &&
           newEventType.userId !== user.id &&
-          !newEventType?.team?.members.filter((membership) => membership.userId === user.id).length
+          !newEventType?.team?.members.find((membership) => membership.userId === user.id) &&
+          !newEventType?.users.find((eventTypeUser) => eventTypeUser.id === user.id)
         ) {
```

Comment:
It definitely deserves a utility fn that can be called like `canMakeChangesToEvent(eventType, user)`. I am sure we must be using similar authorization checks elsewhere. So, either similar fn should exist already or we should make it.

Reply:
Also split each conditional in individual functions and reuse. It's a NIT change but it's welcome.
isUserIdMemberIn